rules/internal/utils/extension_test.go

```diff
@@ -124,6 +124,84 @@ func TestGetOperationInfoNone(t *testing.T) {
 	}
 }
 
+func TestGetOperationInfoResponseType(t *testing.T) {
+	// Set up testing permutations.
+	tests := []struct {
+		testName     string
+		ResponseType string
+		invalid      bool
```

Comment:
super-nit: Commonly preferred to name the variable after the positive rather than the negation, i.e. valid?

Reply:
Done.